projects/plugins/jetpack/extensions/blocks/mailchimp/index.js

```diff
@@ -53,6 +53,16 @@ export const settings = {
 		_x( 'subscription', 'block search term', 'jetpack' ),
 		_x( 'newsletter', 'block search term', 'jetpack' ),
 	],
+	supports: {
+		align: [ 'wide', 'full' ],
+		color: {
+			link: true,
+			gradients: true,
+		},
+		spacing: {
+			padding: true,
+		},
+	},
```

Comment:
I think we can add margin & typography as well.

Reply:
> I think we can add margin & typography as well.

When it comes to adding Typography tool to blocks that have forms in them, we had a discussion with @jeherve on the Form block PR (https://github.com/Automattic/jetpack/pull/22465#issuecomment-1028189343).

> [...] at the [current state](https://github.com/WordPress/gutenberg/blob/add/styles-intro/docs/explanations/architecture/styles.md#current-limitations-of-the-block-supports-api) of "Supports API", each design tool applies to the whole block (in form of style and class attributes of the block wrapper tag).

> This means that the same font-size would apply to all elements of the Form block, including labels, inputs, text areas, etc.... As a result, I suspect that more inconsistencies could arise with other themes.

In other words, adding a block-level Typography tool on blocks like Form or Mailchimp may result in an user experience that doesn't provide predictable results.

Example: In the following recording we can see how font-size applies to the text only and spacing applies to all elements of the form:

https://user-images.githubusercontent.com/25105483/153161043-e5940477-37db-484f-8b73-9824a5aebbd9.mov

Reply:
Okay gotcha! :) Then let's just do margins. :)